packages/charting/src/components/DonutChart/DonutChart.types.ts

```diff
@@ -35,6 +35,11 @@ export interface IDonutChartProps {
    */
   theme?: ITheme;
 
+  /**
+   * color for hover font color
+   */
+  color?: string;
```

Comment:
Can this prop be removed and have the color be specified per-chart using something like `<DonutChart styles={{callOut: {color: 'red'}}} />` instead? The intent here is to keep style-specific properties out of component props and have them be handled by the `styles` prop instead.